skpy/event.py

```diff
@@ -114,7 +118,11 @@ def rawToFields(cls, raw={}):
         fields.update({"userId": SkypeUtils.userToId(res.get("selfLink")),
                        "name": res.get("privateInfo", {}).get("epname"),
                        "capabilities": list(filter(None, res.get("publicInfo", {})
-                                                            .get("capabilities", "").split(" | ")))})
+                                                            .get("capabilities", "").split(" | "))),
+                       "version": res.get("publicInfo", {})
```

Comment:
We're now reading `publicInfo` three times, so we ought to pull that out into its own variable.